components/contextmenu/overview.md

```diff
@@ -96,70 +96,45 @@ In addition to built-in [navigation capabilities]({%slug contextmenu-navigation%
 }
 ````
 
->caption The result from the snippet above, after right clicking the yellow target and hovering the "Advanced" item
-
-![Blazor Context Menu Overview](images/context-menu-overview.png)
-
->caption Component namespace and reference
-
-````CSHTML
-@using Telerik.Blazor.Components
+## Navigate Views
 
-<TelerikContextMenu Data="@MenuItems" @ref="@TheContextMenu">
-</TelerikContextMenu>
+A menu is often used to list pages, views or sections in an application so the user can navigate through them. To do that with a menu, you have two options:
 
-@code {
-    // the context menu is a generic component and its type depends on the model it binds to
-    TelerikContextMenu<ContextMenuItem> TheContextMenu { get; set; }
+* Use the built-in `UrlField` in the [bound data]({%slug contextmenu-data-binding-overview%}) to populate the URLs in the anchors the menu will generate for you if an URL is provided for the given item. An example is available in the beginning of this article.
+* Use a [Template]({%slug contextmenu-templates-overview%}) to generate the desired links (e.g., `NavLink` components) with your own code to enable fine-tuning.
 
-    List<ContextMenuItem> MenuItems { get; set; }
+[Read more about the Blazor Context Menu navigation...]({%slug contextmenu-navigation%})
 
-    protected override void OnInitialized()
-    {
+## Customize per Target
 
-        MenuItems = new List<ContextMenuItem>()
-        {
-            new ContextMenuItem
-            {
-                Text = "Item 1"
-            },
-            new ContextMenuItem
-            {
-                Text = "Item 2"
-            }
-        };
+The same context menu can easily be attached to many targets, or you can use its `ShowAsync(x, y)` method to show it explicitly based on your business logic needs, data and events. Read more in the [Integration]({%slug contextmenu-integration%}) article.
 
-        base.OnInitialized();
-    }
+## Events
 
-    public class ContextMenuItem
-    {
-        public string Text { get; set; }
-        public List<ContextMenuItem> Items { get; set; }
-    }
-}
-````
+The Context Menu generates events that you can handle and further customize its behavior. [Read more about the Blazor Context Menu events...]({%slug contextmenu-events%})
 
+## Templates
 
-## Navigate Views
+You can use the functionality of the built-in templates and customize what is rendered in the items. [Read more about the Blazor Context Menu templates...]({%slug contextmenu-templates-overview%})
 
-A menu is often used to list pages, views or sections in an application so the user can navigate through them. To do that with a menu, you have two options:
+## Data Binding
```

Comment:
Feature sections should be sorted by importance and frequency of usage. Data Binding should come first by all means.